tests/impl.cpp

```diff
@@ -3450,7 +3450,19 @@ result_t test_mm_extract_epi64(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_insert_epi64(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const int64_t *_a = (const int64_t *) impl.mTestIntPointer1;
+    const int64_t insert = (const int64_t ) *impl.mTestIntPointer2;
+    const int imm8 = insert & 1; // insert mod 2
+
+    int64_t d[2];
+    for (int i = 0; i < 2; i++) {
```

Comment:
Eliminate the loop since there are only two elements. Instead, use two assignments such as `d[0] = ...; d[1] = ...;`.